src/tests/App.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import App from "../App";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+
+it("should render Current Notifications", () => {
+  render(<App />);
+  const message = screen.queryByText(/Current Notification/i);
+  expect(message).toBeVisible();
+});
+
+it("renders the pagination bar", () => {
+  render(<App />);
+  const pageButton = screen.getByText("1");
+  userEvent.click(pageButton);
```

Comment:
Is there a reason for these? I believe we can remove lines 15-16 since the pagination renders regardless of a user event

Reply:
Yeah, we can get rid of it, wanted to use userEvents and clicks for when the pagination actually will populate with multiple choices when we have more data